corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -156,6 +156,12 @@
                type="checkbox"
                data-bind="value: uniqueId, checked: $parent.includedFormIds"/>
         <span data-bind="text: name"></span>
+        {% if session_endpoints_enabled %}
+          <label>
```

Comment:
These inputs should be aligned vertically. You could do a table with one column of checkbox + form name and one column of session endpoint id inputs, or you might be able to just float the endpoint id inputs to the right.

Reply:
It is on my list. Once I get this child menu business figured out